src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/illegalidentifiername/Example1.java

```diff
@@ -0,0 +1,40 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalIdentifierName"/>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.illegalidentifiername;
+
+// xdoc section -- start
+public class Example1 {
+  Integer var = 4; // violation, 'Name 'var' must match pattern'
```

Comment:
Omg, this real violation has no sense!!!
Such a terrible design we leaked.
I will create issue to fix violation message to make new violation message: `Name 'var'  matching  illegal name pattern`

Reply:
Do I write "ok until issue #XXXXX"?

Reply:
keep it as it is now, to let users explicity see what type of violation they will have.

Reply:
ok

Reply:
@chrccl , please create issue to change message.
It would be good to improve sooner, as user experience is terrible to read this message.

Reply:
@romani , Sure I create the issue immediately!